plugin/src/android/AppboyPlugin.java

```diff
@@ -10,6 +10,8 @@
 import com.appboy.enums.NotificationSubscriptionType;
 import com.appboy.models.outgoing.AppboyProperties;
 import com.appboy.ui.activities.AppboyFeedActivity;
+import com.appboy.models.outgoing.AttributionData;
+import com.adjust.sdk.AdjustAttribution;
```

Comment:
This is throwing an error on a regular app because Adjust isn't a dependency of our SDK; also I don't believe you're using it, so I think it can just be removed?